test/integration/targets/template/tasks/main.yml

```diff
@@ -120,6 +120,93 @@
         - 'import_as_with_context_diff_result.stdout == ""'
         - "import_as_with_context_diff_result.rc == 0"
 
+# VERIFY trim_blocks
+
+- name: Render a template with "trim_blocks" set to False
+  template:
+    src: trim_blocks.j2
+    dest: "{{output_dir}}/trim_blocks_false.templated"
+    trim_blocks: False
+  register: trim_blocks_false_result
+
+- name: Get checksum of known good trim_blocks_false.expected
+  stat:
+    path: "{{role_path}}/files/trim_blocks_false.expected"
+  register: trim_blocks_false_good
+
+- name: Verify templated trim_blocks_false matches known good using checksum
+  assert:
+    that:
+        - "trim_blocks_false_result.checksum == trim_blocks_false_good.stat.checksum"
+
+- name: Render a template with "trim_blocks" set to True
+  template:
+    src: trim_blocks.j2
+    dest: "{{output_dir}}/trim_blocks_true.templated"
+    trim_blocks: True
+  register: trim_blocks_true_result
+
+- name: Get checksum of known good trim_blocks_true.expected
+  stat:
+    path: "{{role_path}}/files/trim_blocks_true.expected"
+  register: trim_blocks_true_good
+
+- name: Verify templated trim_blocks_true matches known good using checksum
+  assert:
+    that:
+        - "trim_blocks_true_result.checksum == trim_blocks_true_good.stat.checksum"
+
+# VERIFY lstrip_blocks
+
+- name: Check support for lstrip_blocks in Jinja2
+  shell: python -c 'import jinja2; jinja2.defaults.LSTRIP_BLOCKS'
```

Comment:
Use `{{ ansible_python_interpreter }}` instead of `python`.

Reply:
Sometimes our existing tests do things that they shouldn't, such as hard-coding references to `python` instead of using `{{ ansible_python_interpreter }}`.

Reply:
The reason this fixes the test failure is because there is another integration test, `template_jinja2_latest`, which sets up a virtualenv with a newer version of Jinja2. During that test, the shell task calling `python` executes to the Python version inside the virtualenv, which passes the Jinja2 version check. But the Python being used by Ansible is `/usr/bin/python`, which would fail the Jinja2 version test and doesn't support `lstrip_blocks`.

This causes the "Render a template with "lstrip_blocks" set to True" task to run but fail when in should be skipped, making `checksum` undefined in `lstrip_blocks_true_result`.